src/components/CommentUsername/CommentUsername.js

```diff
@@ -33,7 +44,7 @@ const styles = StyleSheet.create({
 const CommentUsername = ({comment, user, isPreviewComment}) => {
   if (isPreviewComment) {
     return (
-      <Text style={[styles.username, styles.previewContainer]}>
+      <Text style={[styles.username, styles.previewContainer]} numberOfLines={1}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `numberOfLines` prop is set to 1, which means that if the username is too long, it will be truncated. This could potentially lead to confusion if there are users with similar starting usernames. Consider adding an ellipsis or some other visual indication that the username has been truncated.
```diff
- <Text style={[styles.username, styles.previewContainer]} numberOfLines={1}>
+ <Text style={[styles.username, styles.previewContainer]} numberOfLines={1} ellipsizeMode='tail'>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->